schema/schema3.cc

```diff
@@ -347,6 +347,7 @@ namespace schema3
     m.fileVersion=minskyVersion;
     
     static_cast<minsky::RungeKutta&>(m)=rungeKutta;
+    m.tmax=tmax;
```

Comment:
This statement shouldn't be needed, as the previous statement will do it for you.